scripts/start.js

```diff
@@ -0,0 +1,19 @@
+import * as dotenv from 'dotenv';
+import Webex from 'webex';
+
+import WebexSDKAdapter from '../src/WebexSDKAdapter';
+
+// Allows convict module to read the corresponding configuration
+dotenv.config({path: `${__dirname}/../.env.development`});
```

Comment:
Can we just have the file loaded from `.env` and then all we have to do is `dotenv.config()`

Reply:
I liked having a dev environment as a separate environment to not mess things up later on, in case we need actual env variables.

Reply:
I can change it to what you want, if you still think it's not necessary.